src/util/util.js

```diff
@@ -109,12 +109,28 @@ function flatten(arr) {
   }, []);
 }
 
+/**
+ * Joins two arrays without duplicates
+ * @param {Array} arr1
+ * @param {Array} arr2
+ */
+function joinArrayUnique(arr1 = [], arr2 = []) {
+  const arr = (arr1 || []).concat(arr2 || []);
```

Comment:
So in case an user of this function passes some non falsy parameters, we are taking an empty array instead and use this for the concat. What is the benefit of that?

Reply:
it works as a safe guard, because sometimes we pass null or undefined values without knowing. Also another benefit is that it removes duplicates

Reply:
yep but shouldn't the safe guard be something like 
```javascript
Array.isArray(arr1) || []
```
?

Reply:
I made the change, can you validate it?